frontend/pendingTranslations.ftl

```diff
@@ -2,25 +2,14 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-menu-upgrade-button = Upgrade
-menu-toggle-open = Open menu
-menu-toggle-close = Close menu
 nav-dashboard = Dashboard
 nav-phone = Phone Number
-nav-settings = Settings
-nav-support = Help and Support
-nav-sign-out = Sign Out
-nav-contact = Contact Us
 nav-duo-description = Switch dashboards
 nav-duo-email-mask-alt = Email masks
 nav-duo-phone-mask-alt = Phone masks
 
 setting-api-key-copied-alt = Click to copy
 
-waitlist-control-country-label-2 = What country or region do you live in?
-
-fx-desktop-2 = { -brand-name-firefox } for Desktop
-fx-mobile-2 = { -brand-name-firefox } for Mobile
 fx-containers = { -brand-name-firefox } Containers
```

Comment:
@lloan Am I correct in thinking that this string does not get used anywhere in the codebase, and thus can be removed as well?